content/vault/v1.19.x/content/docs/updates/important-changes.mdx

```diff
@@ -418,3 +418,19 @@ to the billing start date for the cluster.
 To avoid issues and unexpected behavior with Vault Community, always provide a
 valid `start_time` value when calling `/sys/internal/counters/activity` or
 upgrade to v1.20.x.
+
```

Comment:
Please make sure to update /vault/global/partials/important-changes/summary-tables/1_19.mdx as well. You can copy the row used in 1_20.mdx, just make sure to update the link to use "v1.19.x" as the version number